<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMD_MainWindow</class>
 <widget class="QMainWindow" name="SMD_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Storage Map Designer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="StorageMap_View">
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foregroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="acLoadGraphML"/>
    <addaction name="acSaveGraphML"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="acFitToPage"/>
    <addaction name="acZoomIn"/>
    <addaction name="acZoomOut"/>
    <addaction name="acGrid"/>
    <addaction name="acSnapToGrid"/>
    <addaction name="acBBox"/>
   </widget>
   <widget class="QMenu" name="menuGraph_Edit">
    <property name="title">
     <string>Graph Edit</string>
    </property>
    <addaction name="acAddNode"/>
    <addaction name="acAddEdge_direct"/>
    <addaction name="acAddEdge_reverse"/>
    <addaction name="acAddEdge"/>
    <addaction name="acDelMultiEdge"/>
    <addaction name="acReverseEdges"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuGraph_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolMain">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acFitToPage"/>
   <addaction name="acZoomIn"/>
   <addaction name="acZoomOut"/>
   <addaction name="acGrid"/>
   <addaction name="acSnapToGrid"/>
   <addaction name="acBBox"/>
   <addaction name="acInfoRails"/>
   <addaction name="acMainRail"/>
  </widget>
  <widget class="QToolBar" name="toolEdit">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acAddNode"/>
   <addaction name="acAddEdge_direct"/>
   <addaction name="acAddEdge_reverse"/>
   <addaction name="acAddEdge"/>
   <addaction name="acDelMultiEdge"/>
   <addaction name="acReverseEdges"/>
  </widget>
  <widget class="QDockWidget" name="dkObjectProps">
   <property name="windowTitle">
    <string>Object Props</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTableView" name="tvObjectProps">
       <property name="minimumSize">
        <size>
         <width>325</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView::item:focus
{
border : 2px solid blue;
/*color: white;*/
/*selection-color: black;
selection-background-color: red;*/
}
</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkView">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="lbGridSize">
       <property name="text">
        <string>Grid size:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="sbGridSize">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="CActionButton" name="abGrid">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="Action" stdset="0">
        <string>acGrid</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CActionButton" name="abSnapToGrid">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="Action" stdset="0">
        <string>acSnapToGrid</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="CActionButton" name="abBBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="Action" stdset="0">
        <string>acBBox</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="acFitToPage">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_fit.png</normaloff>:/images/zoom_fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit To Page</string>
   </property>
   <property name="toolTip">
    <string>Fit To Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="acZoomIn">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_in.png</normaloff>:/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="acZoomOut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_out.png</normaloff>:/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="acGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/grid.png</normaloff>:/images/grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Grid</string>
   </property>
   <property name="shortcut">
    <string>#</string>
   </property>
  </action>
  <action name="acBBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/bbox.png</normaloff>:/images/bbox.png</iconset>
   </property>
   <property name="text">
    <string>BBox</string>
   </property>
   <property name="toolTip">
    <string>BBox</string>
   </property>
  </action>
  <action name="acLoadGraphML">
   <property name="text">
    <string>Load GraphML</string>
   </property>
   <property name="toolTip">
    <string>Load GraphML</string>
   </property>
  </action>
  <action name="acSaveGraphML">
   <property name="text">
    <string>Save Graph ML</string>
   </property>
   <property name="toolTip">
    <string>Save Graph ML</string>
   </property>
  </action>
  <action name="acAddNode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_node.png</normaloff>:/images/add_node.png</iconset>
   </property>
   <property name="text">
    <string>Add node mode</string>
   </property>
   <property name="toolTip">
    <string>Add node mode</string>
   </property>
  </action>
  <action name="acAddEdge">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_edge.png</normaloff>:/images/add_edge.png</iconset>
   </property>
   <property name="text">
    <string>Add edges</string>
   </property>
   <property name="toolTip">
    <string>Add edges for selected nodes</string>
   </property>
  </action>
  <action name="acAddEdge_direct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_edge_1-2_off.png</normaloff>
     <normalon>:/images/add_edge_1-2.png</normalon>:/images/add_edge_1-2_off.png</iconset>
   </property>
   <property name="text">
    <string>Add edges direct mode</string>
   </property>
   <property name="toolTip">
    <string>Add edges in a direct order (1 --&gt; 2)</string>
   </property>
  </action>
  <action name="acAddEdge_reverse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_edge_2-1_off.png</normaloff>
     <normalon>:/images/add_edge_2-1.png</normalon>:/images/add_edge_2-1_off.png</iconset>
   </property>
   <property name="text">
    <string>Add edges reverse mode</string>
   </property>
   <property name="toolTip">
    <string>Add edges in a reverse order (1 &lt;-- 2)</string>
   </property>
  </action>
  <action name="acDelMultiEdge">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/del_edge.png</normaloff>:/images/del_edge.png</iconset>
   </property>
   <property name="text">
    <string>Delete multiple edge</string>
   </property>
   <property name="toolTip">
    <string>Delete multiple edge</string>
   </property>
  </action>
  <action name="acReverseEdges">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/reverse_edge.png</normaloff>:/images/reverse_edge.png</iconset>
   </property>
   <property name="text">
    <string>Reverse selected edges</string>
   </property>
   <property name="toolTip">
    <string>Reverse selected edges</string>
   </property>
  </action>
  <action name="acSnapToGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/snap_to_grid.png</normaloff>:/images/snap_to_grid.png</iconset>
   </property>
   <property name="text">
    <string>Snap to grid</string>
   </property>
   <property name="toolTip">
    <string>Snap to grid</string>
   </property>
  </action>
  <action name="acInfoRails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/info_rails.png</normaloff>:/images/info_rails.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide info rails</string>
   </property>
   <property name="toolTip">
    <string>Show/hide info rails</string>
   </property>
  </action>
  <action name="acMainRail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main_rail.png</normaloff>:/images/main_rail.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide main rail</string>
   </property>
   <property name="toolTip">
    <string>Show/hide main rail</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CActionButton</class>
   <extends>QPushButton</extends>
   <header>actionbutton</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
